app/views/template_inputs/_form.html.erb

```diff
@@ -20,6 +20,8 @@
         <%= checkbox_f f, :advanced, :disabled => @template.locked? %>
         <%= textarea_f f, :options, :rows => 3, :class => 'user_input_type', :help_inline => _("A list of options the user can select from. If not provided, the user will be given a free-form field"),
                                     :disabled => @template.locked?, wrapper_class: "form-group input-options-#{f.index || f.object.id}#{' hide' if f.object.value_type != 'plain'}"  %>
+        <%= textarea_f f, :default, :rows => 1, :class => 'user_input_type',
```

Comment:
should this be input instead of textarea? it's probably single string value

Reply:
Thanks, changed.